bindings/python/iota_sdk/utils.py

```diff
@@ -94,24 +94,60 @@ def compute_alias_id(output_id: OutputId) -> HexStr:
             'outputId': output_id
         })
 
+    @staticmethod
+    def compute_foundry_id(alias_id: str, serial_number: int, token_scheme_kind: int) -> HexStr:
+        """Computes the foundry id.
+        """
+        return _call_method('computeNftId', {
+            'aliasId': alias_id,
+            'serialNumber': serial_number,
+            'tokenSchemeKind': token_scheme_kind
+        })
+
+    @staticmethod
+    def compute_inputs_commitment(inputs) -> HexStr:
+        """Calculate the input commitment from the output objects that are used as inputs to fund the transaction.
+        """
+        return _call_method('computeInputsCommitment ', {
+            'inputs': inputs
+        })
+
+    @staticmethod
+    def compute_storage_deposit(output, rent) -> HexStr:
+        """Calculates the required storage deposit of an output.
+        """
+        return _call_method('computeStorageDeposit  ', {
+            'inputs': inputs
+        })
+
     @staticmethod
     def compute_nft_id(output_id: OutputId) -> HexStr:
         """Computes the NFT id for the given NFT output id.
         """
         return _call_method('computeNftId', {
             'outputId': output_id
         })
+        
+    @staticmethod
+    def compute_output_id(transaction_id: HexStr, index: int) -> HexStr:
+        """Returns the output ID from transaction id and output index.
+        """
+        return OutputId.from_string(_call_method('computeOutputId', {
+            'transactionId': transaction_id,
+            'index': serial_number,
+        }))
```

Comment:
This looks a bit stupid `OutputId(transaction_id, index)` does the job already
(also return type should be OutputId and `serial_number` should be `index`)

Reply:
yea i saw that, but the whole point of these bindings was not to do computations inside them right? Else i could add a lot more to not need to use these utils (every compute_ could be done in bindings)

Dunno what to do with it. @thibault-martinez when do you think bindings should be used vs local generation?

Reply:
The difference here is that there is nothing to "compute", it's literally just a concatenation, no hash involved or whatever. So I guess I would tend to agree with Thoralf on this one. How do we construct an OutputId if we don't use the MI though? Do we actually have `OutputId(transaction_id, index)`?

Reply:
We have it in iota_sdk/types/output_id, but therse a bunch of checks in place, conversion to bytes and yes, concatenation. 
What is your defenition of computation? in the end its all concatenation of bytes ;)
I would have to add the same code (read: duplicate) in nodejs

Reply:
I guess we could also call `compute_output_id()` in the init method of `OutputId` so nothing is duplicated if you think that's better

Reply:
Hey i dont think its better :D but we should not go down that slippery road of "just a small calculation" haha
ill update it

Reply:
couldnt add the util call because of circular references. lets leave it as is for now

Reply:
Well at least fix this `also return type should be OutputId and serial_number should be index`